scripts/fantom_apy.py

```diff
@@ -25,4 +29,4 @@ def fetch_apy(vault_address):
         raise Exception("run on fantom")
     samples = get_samples()
     vault = VaultV2.from_address(vault_address)
-    print(json.dumps(dataclasses.asdict(vault.apy(samples)), indent=2))
\ No newline at end of file
+    print(json.dumps(dataclasses.asdict(vault.apy(samples)), indent=2))
```

Comment:
did you keep this print intentionally or should it be `logger.debug` ?

Reply:
uh. I didn't mean to do anything in this file, perhaps I messed up the rebase. Will revert this file to its current state on master

Reply:
Aside from the sorting of imports and addition of the sentry tag, I just left everything else as it was